src/SignhostAPIClient.Tests/SignHostApiClient.cs

```diff
@@ -19,11 +19,53 @@ public class SignHostApiClientTests
+
+			// Act
+			bool validatePostback = SignHostApiClient.IsPostbackChecksumValid(falsechecksum, sharedSecret);
+
+			// Assert
+			validatePostback.ShouldBeEquivalentTo(false);
+		}
+
+		[Fact]
+		public void when_IsPostbackChecksumValid_is_called_with_invalid_postback_format_then_exception_is_thrown()
+		{
+			// Arrange
+			string sharedSecret = "UwMKSuFPtydaZHpNzVyJDfgX7f1wMf6F";
+			string falsepostback = @"{""I'm not a valid postback"":""I'm still not a valid postback""}";
+			//string falsepostback = "I'm not a valid postback";
+
+			// Act
+			Action validatePostback = () => SignHostApiClient.IsPostbackChecksumValid(falsepostback, sharedSecret);
```

Comment:
Using `Action` is not consistent with other tests, if the format is invalid shouldn't the `IsPostbackChecksumValid` just return `false`?